src/MembershipsSDKAdapter.js

```diff
@@ -31,6 +31,23 @@ const MAX_MEMBERSHIPS = 1000;
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L6}
  */
 
+/**
+ *
+ * @param {object} members Members object from meeting, keyed by ID
+ * @returns {Array} Sorted (by displayName) list of active users in a meeting
+ */
+function sortMembers(members) {
+  return members.sort((member1, member2) => {
```

Comment:
one approach to having a single return:
```js
return members.sort((member1, member2) => (
  member1.isSelf ? -1 : // current user comes first
  member2.isSelf ? 1 :
  !member1.name ? 1 : // empty names come last
  !member2.name ? -1 :
  member1.name.localeCompare(member2.name)  // alphabetical order
);
```

Reply:
@lalli-flores there is an issue with this approach, the linter complains about nested ternary operators. Would it be ok to disable that check for this instance?

Reply:
It should be ok to disable the linter for this one line but not at the module level